tools/sggolangcilint/tools.go

```diff
@@ -14,30 +15,57 @@ import (
 	"go.einride.tech/sage/sgtool"
 )
 
-const version = "1.43.0"
+const version = "1.44.0"
 
 // nolint: gochecknoglobals
 var commandPath string
 
 //go:embed golangci.yml
-var defaultConfig string
+var defaultConfig []byte
 
 func Command(ctx context.Context, args ...string) *exec.Cmd {
 	sg.Deps(ctx, PrepareCommand)
 	return sg.Command(ctx, commandPath, args...)
 }
 
-func RunCommand(ctx context.Context) *exec.Cmd {
-	configPath := sg.FromWorkDir(".golangci.yml")
-	cmd := Command(ctx)
-	if _, err := os.Stat(configPath); errors.Is(err, os.ErrNotExist) {
-		configPath = sg.FromToolsDir("golangci-lint", ".golangci.yml")
-		if err := os.WriteFile(configPath, []byte(defaultConfig), 0o600); err != nil {
-			panic(err)
+// Run GolangCI-Lint in every Go module from the root of the current git repo.
+func Run(ctx context.Context) error {
+	defaultConfigPath := sg.FromToolsDir("golangci-lint", ".golangci.yml")
+	if err := os.MkdirAll(filepath.Dir(defaultConfigPath), 0o755); err != nil {
+		return err
+	}
+	if err := os.WriteFile(defaultConfigPath, defaultConfig, 0o600); err != nil {
+		panic(err)
```

Comment:
I vote return err here since we do elsewhere.

Reply:
Ooops! Thanks!